libcxxabi/src/demangle/ItaniumDemangle.h

```diff
@@ -4468,7 +4468,9 @@ Node *AbstractManglingParser<Derived, Alloc>::parseType() {
         return nullptr;
       if (!consumeIf('_'))
         return nullptr;
-      return make<BitIntType>(Size, Signed);
+      // The FE expects this to be available for Substitution
```

Comment:
Perhaps naive question, but by `FE` do you mean the Front-end? If so, it's worth spelling to avoid confusion.